charter/index.html

```diff
@@ -180,7 +180,7 @@ <h2>Motivation and Background</h2>
         They are</span> largely based on existing open web standards, which they extend to give priority to individual and group autonomy.</p>
         
         <p>The <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> has been incubating <a href="https://solidproject.org/TR/">specifications</a> since 2018 towards that end, i.e., "to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces."
-        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/groups/cg/credentials/">Credentials Community Group (CCG)</a>, the <a href="https://www.w3.org/community/fed-id/">Federated Identity Community Group</a>, the <a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator Community Group</a>, and the <a href="https://www.w3.org/groups/cg/wicg/">Web Platform Incubator Community Group (WICG)</a>).</span></p>
+        <span class=new>It has regularly sought cross-fertilization with the aforementioned communities, among others (e.g., the <a href="https://www.w3.org/groups/cg/credentials/">Credentials Community Group (CCG)</a>, the <a href="https://www.w3.org/groups/wg/fedid/">Federated Identity Working Group</a>, and the <a href="https://www.w3.org/groups/cg/socialcg/">Social Web Incubator Community Group</a>).</span></p>
```

Comment:
If we talk about the past, it might make sense to leave FedId CG since the WG started relatively recently.

Reply:
That is a good point. I have reverted that change in https://github.com/solid/solid-wg-charter/pull/78/commits/8d58e854d184b481ce0be5ab97cdf526486481b8